### DISCUSSION TITLE
Is there a way to regroup the items in the outline?

### DISCUSSION THREAD

**Discussion by dhfreinkel** [codefori/vscode-ibmi#918] [IBM i] [answered]
I hav not been able to figure the sequence of the outline. Is there a way to get it to sort the items by group, or in the sequence of the definitions?

Example: 
- files first
-  sub-procedures
- Then data structures
- then variables

and so on.

Thanks.
TIA, Darryl.

**Comment by worksofliam** [comment, marked as answer]
The outline view in VS Code appears in order of definition by default

You are able to change the sort from the outline view also:

<img width="572" alt="image" src="https://user-images.githubusercontent.com/3708366/197886829-6bf5b998-3839-4128-998e-f56967c69672.png">

**Comment by dhfreinkel** [comment]
How did you get the popup?

I tried to find something like it.

 

I’ll try in the morning again.

 

Thank you.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562

**Comment by dhfreinkel** [comment]
Can you add entry parameters or Dcl-pi variables?

 

Thanks

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562

**Comment by dhfreinkel** [comment]
Thanks, Found the place.

 

Darryl Freinkel

A4G

Telephone: 770.321.8562 Mobile: 678.355.8562
